src/tuple.h

```diff
@@ -100,7 +101,7 @@ int tuple_decoder__init(struct tuple_decoder *d,
  * parameters format this is the value contained in the first byte of the tuple
  * header.
  */
-unsigned tuple_decoder__n(struct tuple_decoder *d);
+unsigned long tuple_decoder__n(struct tuple_decoder *d);
```

Comment:
Switched to `unsigned long` here and elsewhere because that's the smallest `int` flavor that's guaranteed to be at least 4 bytes wide.